test/parallel/test-handle-wrap-isrefed.js

```diff
@@ -3,99 +3,128 @@
 }
 
 
 // pipe
 {
-  const assert = makeAssert('hasRef() not working on pipe_wrap');
   const Pipe = process.binding('pipe_wrap').Pipe;
   const handle = new Pipe();
-  assert(Object.getPrototypeOf(handle).hasOwnProperty('hasRef'), true);
-  assert(handle.hasRef(), true);
+  strictEqual(Object.getPrototypeOf(handle).hasOwnProperty('hasRef'),
+              true, 'pipe_wrap: hasRef() missing');
+  strictEqual(handle.hasRef(),
+              true, 'pipe_wrap: not initially refed');
   handle.unref();
-  assert(handle.hasRef(), false);
+  strictEqual(handle.hasRef(),
+              false, 'pipe_wrap: unref() ineffective');
   handle.ref();
-  assert(handle.hasRef(), true);
-  handle.close(common.mustCall(() => assert(handle.hasRef(), false)));
+  strictEqual(handle.hasRef(),
+              true, 'pipe_wrap: ref() ineffective');
+  handle.close(common.mustCall(() =>
+      strictEqual(handle.hasRef(),
+                  false, 'pipe_wrap: not unrefed on close')));
 }
 
 
 // tcp
 {
-  const assert = makeAssert('hasRef() not working on tcp_wrap');
   const net = require('net');
   const server = net.createServer(() => {}).listen(0);
-  assert(Object.getPrototypeOf(server._handle).hasOwnProperty('hasRef'), true);
-  assert(server._handle.hasRef(), true);
-  assert(server._unref, false);
+  strictEqual(Object.getPrototypeOf(server._handle).hasOwnProperty('hasRef'),
+              true, 'tcp_wrap: hasRef() missing');
+  strictEqual(server._handle.hasRef(),
+              true, 'tcp_wrap: not initially refed');
+  strictEqual(server._unref,
+              false, 'tcp_wrap: _unref initially incorrect');
   server.unref();
-  assert(server._handle.hasRef(), false);
-  assert(server._unref, true);
+  strictEqual(server._handle.hasRef(),
+              false, 'tcp_wrap: unref() ineffective');
+  strictEqual(server._unref,
+              true, 'tcp_wrap: _unref not updated on unref()');
   server.ref();
-  assert(server._handle.hasRef(), true);
-  assert(server._unref, false);
-  server._handle.close(
-      common.mustCall(() => assert(server._handle.hasRef(), false)));
+  strictEqual(server._handle.hasRef(),
+              true, 'tcp_wrap: ref() ineffective');
+  strictEqual(server._unref,
+              false, 'tcp_wrap: _unref not updated on ref()');
+  server._handle.close(common.mustCall(() =>
+      strictEqual(server._handle.hasRef(),
+                  false, 'tcp_wrap: not unrefed on close')));
 }
 
 
 // timers
 {
-  const assert = makeAssert('hasRef() not working on timer_wrap');
   const timer = setTimeout(() => {}, 500);
   timer.unref();
-  assert(Object.getPrototypeOf(timer._handle).hasOwnProperty('hasRef'), true);
-  assert(timer._handle.hasRef(), false);
+  strictEqual(Object.getPrototypeOf(timer._handle).hasOwnProperty('hasRef'),
+              true, 'timer_wrap: hasRef() missing');
+  strictEqual(timer._handle.hasRef(),
+              false, 'timer_wrap: unref() ineffective');
```

Comment:
Yes there was; it's just a line further up.

Apparently timers don't have `_handle` defined prior to `unref()` being called, so we cannot check `hasRef()` exists, nor assert an initially `refed` state for them. Instead we check the existence of `hasRef()` and the `unrefed` state after already calling `unref()`.